qualang_tools/results/results.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, job, data_list, mode="wait_for_all"):
     def is_processing(self):
         """
         Returns True while the program is processing. Used for live plotting.
+        The time at which the processing starts is stored in `self.start_time` to feed `progress_counter()`.

```

Comment:
I've just noticed you added this. I'm ok with having this and not a getter function (although I think it's nicer).
But I would rephase to:

The time at which this method is first called is stored in `self.start_time`. It can be used to for `progress_counter()`.